### PR TITLE
chore: add metadata documentation

### PR DESCRIPTION
**Pull request by DaRaFF**

### CODE REVIEW

File: core/livingdocs-server/metadata.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Scope
+
+This documentation assumes that you have setup a channel and design within Livingdocs.
+You also know what an environment and a channel config is.
+
+## Introduction to metadata
+
+Every document(article, page) can have metadata. A metadata object consist of key-value pairs
+and the key is unique.
```
Comment (marcbachmann) [inline core/livingdocs-server/metadata.md]: key-value pairs where the key is unique per document.

File: core/livingdocs-server/metadata.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  # Deprecated
+  # Is the same as `onUpdate` without documentVersion as parameter
+  set: (newValue, oldValue, config) ->
+    ...
+```
+
+# Example: Use a Standard Plugin
+
+A standard plugin is stored in `plugins/metadata` and will automatically
+loaded during the server start.
```
Comment (marcbachmann) [inline core/livingdocs-server/metadata.md]: automatically load during

File: core/livingdocs-server/metadata.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+If you want to define and use a default plugin, you just have to define
```
Comment (marcbachmann) [inline core/livingdocs-server/metadata.md]: Please remove `just`. It might offend the reader if he doesn't understand the setup right away.

Comment (marcbachmann) [inline core/livingdocs-server/metadata.md]: replace the second `define` with `declare`

**Comment by DaRaFF** [comment]
@marcbachmann @masone pls review

File: core/livingdocs-server/metadata.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+  "url": "/test/doctor-who",
+  "seo": {
+    "title": "BBC TV Show - Doctor Who",
+    "description": "Best TV show of all times",
+    "newsKeywords": "tv bbc",
+    "excludeFromSpiders": false
+  }
+}
+```
+Metadata can be seen as a public API for a document. Frontends usually access on metadata
```
Comment (masone) [inline core/livingdocs-server/metadata.md]: Good point. Eg. renaming a metadata field likely results in a change in the frontend. Never thought of it that way

**Comment by masone** [comment]
How does it play with https://github.com/upfrontIO/livingdocs/blob/master/public/livingdocs-editor/configurable-metadata-screen.md - any overlaps? Should we extract generic things to a concept instead of documenting it in the server / editor?

**Comment by DaRaFF** [comment]
@masone There is a small overlap, but I suggest I will move the server part of the editor documentation into the server documentation. I still don't understand the different sections of the documentation. For the moment I would just create a cross reference between the frontend/backend part for metadata. Maybe we can extract some concept part later.

**Comment by masone** [comment]
If it describes both server and editor, consider moving it to guides (merging server and editor) and extracting the concept

**Comment by peyerluk** [comment]
@masone Good to merge? Or are further changes required?

**Comment by masone** [comment]
When I interpret the comments above correctly, @DaRaFF wanted to do further work on it

**Comment by DaRaFF** [comment]
@masone maybe it's cool to merge and improve it later?

**Comment by masone** [comment]
I think it should take you about 30 mins to integrate your additions to the existing one. If you can spare the time, I'd welcome no duplication

**Comment by DaRaFF** [comment]
@masone I didn't see duplication. My PR describes how metadata work on the server generally and @gabrielhase's document describes mainly the editor side with a specific example. 

I will add a cross reference to both articles. I guess then it should be fine.

**Comment by masone** [comment]
Thanks for spending some additional time checking! 🚢

**Comment by DaRaFF** [comment]
@masone ok, I've improved the documentation. Please make a final review.

**Comment by masone** [comment]
Thanks, good to go! Very valuable addition

**Pull request merged**